test/cmd.js

```diff
@@ -467,6 +506,60 @@ describe('express(1)', function () {
     })
   })
 
+  describe('--no-view', function () {
+    var ctx = setupTestEnvironment(this.fullTitle())
+
+    it('should create basic app with no view engine', function (done) {
+      run(ctx.dir, ['--view', 'none'], function (err, stdout) {
```

Comment:
It works mostly by coincidence. `--no-view` ultimately maps to `program.view === 'none'`, but there's nothing that validates that `program.view` is set to a supported value with the `--view=<value>` parameter. So passing in `--view=none` also sets the `view` to `none`. I think the only way to avoid something like that occurring (if it were changed to `foo`, it would work for `--view=foo`) is with active validation that the parameter passed in is one of the suspected values. Perhaps it'd be marginally less surprising if I change `none` to `no-view`, although that pretty much just hides the issue.